tests/all-tests.js

```diff
@@ -1,3 +1,9 @@
-require("./js/requestAware/requestAware-tests");
+// TODO: Testing in this order works, alphabetical order does not.  Review with Antranig.
```

Comment:
Previously there were errors, but sadly I didn't write them down.  Alphabetical order now works, so let's leave it for now.